### DISCUSSION TITLE
Cloning repositories but forgot passphrase key

### DISCUSSION THREAD

**Discussion by mojallo** [community/community#43739] [New to GitHub]
I made a new repository and I am trying to clone it onto my local machine using gitclone git@github.com:mojallo/odin-recipes.git, but it keeps asking me for a passphrase that I forgot, how do I reset this or solve this issue. Thank you.

**Comment by GabyCT** [comment]
If you lose your SSH key passphrase, there's no way to recover it. You'll need to [generate a brand new SSH keypair](https://docs.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) or [switch to HTTPS cloning](https://docs.github.com/en/github/getting-started-with-github/about-remote-repositories#cloning-with-https-urls) so you can use a personal access token instead.

**Reply by mojallo** [reply]
Thank you so much!!!
